spec/rspec_puppet_facts_spec.rb

```diff
@@ -900,25 +900,41 @@
     end
 
     it 'adds a simple fact and value' do
-      add_custom_fact 'root_home', '/root'
-      expect(subject['redhat-7-x86_64']['root_home']).to eq '/root'
+      add_custom_fact :root_home, '/root'
```

Comment:
You can keep the old test and assert that it's added as a symbol instead. Then add a new test that adds it as a symbol.

Reply:
Updated and new test added!